docs/docsite/rst/scenario_guides/guide_scaleway.rst

```diff
@@ -0,0 +1,231 @@
+Using Scaleway with Ansible
```

Comment:
Please add an anchor above the title so we can use `:ref:`-style links to this page. Syntax is `.. _link_name:` - in this case `.. _guide_scaleway:` would work well. Also, please follow the heading guidelines - page headers should use ******** above and below the text.